textBasedAdventure.js

```diff
@@ -1,5 +1,246 @@
 const readline1 = require('readline-sync')
 
-let nameInput = readline.question("Enter your name: ")
+let nameInput = readline1.question("Enter your name: ")
+    console.log(`I'm pleased to meet you ${nameInput}!`)
+    let askOen =  readline1.keyIn("Im ready to go around the world Lalalala, are you set?")
+    console.log(` Nice! can't wait to show you cool fun Fact we learnt in each country`)
+    console.log("Each time we arrive at our destination we will need to Unscramble the word ");
+    console.log("you will have up to three tries to get it RIGHT");
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+function tryInGame () {
+    let question = readline1.keyInYN(`Do you want to fly around the world Lalalala`)
+    if( question === true) {
+         playGame()
+     }else {
+        leaveGame()
+    }
+}
+tryInGame();
+
+function playGame () {
+   
+    console.log("It will be an amazing Journey");
+    console.log(" You will Unscramble the letter provided to find the correct country so we can advance you throughout your amazing journey");
+    console.log("Each country you discover you will learn a secret fun fact about that country");
+
+    if(readline1.keyInYN("Ready to hit the road!")) {
+        firstCountry();
+
+    } else {
+        leaveGame();
+    }
+
+}
+
+function firstCountry ()  {
+     let question;
+    
+     while (question !== "spain" && question !== "Spain") {
+         question = readline1.question(` What Country is "pains" ?`).trim().toLowerCase();    
+     }
+     if (question === "spain") {
+        console.log(` Spain is correct! Pablo Picasso was a spanish painter, he helped develop Cubism
+        a newly- invented style in which objects were represented by cubes, cones and cylinders.`) 
+     
+     }else if ( question !== "spain") {
```

Comment:
Because of the while loop we'll never reach this code because we only get out of the while loop when question is spain. This is true for all these questions.